content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,61 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing alternatives to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+project</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a valid
+URL (or clone a <abbr>DVC repository</abbr> with one previously configured), and
+use any DVC command that requires the remote (see `dvc pull`, `dvc fetch`,
+`dvc push`).
+
+## URL format for Google Drive remotes
+
+A GDrive remote URLs is needed for the `dvc remote add` command. It can be
+constructed with a _base_, and an **optional** _path_ to a folder i.e.
+`gdrive://<base>/path/to/folder`. The base can be one of:
+
+1. `root` - indicates your topmost Google Drive directory.
+
+```dvc
+$ dvc remote add -d mygdroot gdrive://root
+$ dvc remote add mygdfolder gdrive://root/path/to/folder
+```
+
+2. Google Drive _Folder ID_
+
+To obtain the unique _Folder ID_ in question, navigate into that folder in your
+web browser, and find it in the address bar. For example, for
+`https://drive.google.com/drive/u/0/folders/0AIac4JZqHhKmUk9PDA`:
+
+```dvc
+$ dvc remote add mygdfolder gdrive://0AIac4JZqHhKmUk9PDA
+$ dvc remote add mygdsubfolder \
+                 gdrive://0AIac4JZqHhKmUk9PDA/sub/path
+```
+
+Note that
+[Shared drives](https://developers.google.com/drive/api/v2/about-shareddrives)
+can only be referenced by _Folder ID_.
+
+3. `appDataFolder` -
+   [special hidden folder](https://developers.google.com/drive/api/v2/appdata)
+   that can be used to store application-specific data.
+
+```dvc
+$ dvc remote add mygdappata gdrive://appDataFolder
+```
+
+## Configure custom Google Cloud project & app (recommended)
+
+Optionally, follow this guide to setup your own Google Cloud project and
+generate OAuth credentials for your <abbr>DVC projects<abbr> to connect to
+Google Drive. Doing so avoids sharing our default GCP project/app with every
+other casual user of GDrive remotes out there. This gives you full control over
+Google API usage and rate limits, and ensures optimal performance to access your
+Google Drive with DVC. This is highly recommended for heavy and enterprise use.
```

Comment:
> This is highly recommended for heavy and enterprise use.

Not sure about enterprise. Let's just keep heavy, frequent, or something? Should be enough?

The whole paragraph is too technical and complicated. Can we explain it in some simple way?

Reply:
Simple cases:

* limits (Retying something ... message should be visible in logs), possibility to ask Google to extend limits independently - they answer quickly
* use service accounts (e.g. some automation, CI/CD, etc)

*

Reply:
Updated with a bullet list, please check.